cmd/kubelet/app/plugins.go

```diff
@@ -106,9 +106,14 @@ func ProbeNetworkPlugins(pluginDir, cniConfDir, cniBinDir string) []network.Netw
 	if cniConfDir == "" {
 		cniConfDir = pluginDir
 	}
+
+	binDir := cniBinDir
+	if binDir == "" {
```

Comment:
The plan is to deprecate `network-plugin-dir` flag. And this PR is mainly fixing the regression.